tests/apollo/util/skvbc.py

```diff
@@ -227,6 +229,31 @@ async def prime_for_state_transfer(
                 verify_checkpoint_persistency=persistency_enabled)
 
             return client, known_key, known_val
+    
+    async def write_with_multiple_clients_for_state_transfer(
```

Comment:
Was write_with_multiple_clients_for_state_transfer intended to send requests from multiple clients or single client?

Reply:
Yes celine its supposed to send multiple client requests.

Reply:
How does it complicate the code?
send_concurrent_ops is used in 3-4 places, so any refactoring is not a big deal.

- replace 2nd parameter num_ops with a callable function as a parameter:
`send_concurrent_ops(self, exit_predicate, ...`

- replace
`while sent < num_ops:`
with
`while not exit_predicate():`

That's all, all you need is to define the functions (lambda) in the caller, 1 liner.
I don't see any complications.
Code reuse is important...